### ISSUE TITLE
Belief MCTS page missing in docs

### ISSUE THREAD

**Issue by Omastto1** [JuliaPOMDP/MCTS.jl#81]
The main page of documentation lists the Belief MCTS page. However, the page does not exist.

**Comment by zsunberg** [comment]
Yeah... need to figure out the best way to run Documenter on all the packages to rebuild it. Should fix this at the same time as JuliaPOMDP/POMDPs.jl#360
